CommonFiles/KeyboardFunctions.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  KeyboardFunctions.swift
+//  iOSPracticals
+//
+//  Created by Yagnik Bavishi on 26/04/22.
+//
+
+import UIKit.UIViewController
+
+// MARK: - UIViewController
```

Comment:
give file name as UIViewExtension you are creating extension for UIViewController